server/game/cards/05-DT/DeepDive.js

```diff
@@ -0,0 +1,25 @@
+const Card = require('../../Card.js');
+
+class DeepDive extends Card {
+    //Play: Discard the top card of your opponent's deck. Put a creature that shares a house with the discarded card on the bottom of its owner's deck.
+    setupCardAbilities(ability) {
+        this.play({
+            condition: (context) => context.player.opponent.deck.length > 0,
+            gameAction: ability.actions.discard((context) => ({
+                target: context.player.opponent.deck.slice(0)
```

Comment:
You are going to discard the whole deck :) :)

Either:
context.player.opponent.deck.slice(0)
or
context.player.opponent.deck[0]

Reply:
am I missing something gain because I think I have it as .slice(0)?

Reply:
Meant .slice(0,1)